oss-cloud-backend/services/databaseService.js

```diff
@@ -57,3 +57,8 @@ module.exports.getAllContributions = () => mysql.query('SELECT * FROM contributi
 
 module.exports.getContributorPullRequests = (username) => mysql
   .query('SELECT * FROM contributions WHERE author=?', [username]);
+
+
+module.exports.updateContributionStatus = (status, owner, repo, number) => mysql
+  .query('UPDATE contributions SET status=? WHERE (owner=?) AND (repo=?) AND (number=?)',

```

Comment:
I would suggest to introduce surrogate auto incremented primary key for contributions and contributors and update all queries accordingly.

Reply:
Surrogate keys added.